### PR TITLE
chore(safari): add supported languages

### PR DESCRIPTION
**Pull request by rofe**

### CODE REVIEW

File: src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "Extension AEM Sidekick";
```
Comment (kptdobe) [inline src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings:2]: I would rather say:
```suggestion
"CFBundleDisplayName" = "AEM Sidekick extension";
```

Are those translations managed by the translation team ? If yes, then current version is ok too.

File: src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+/* Bundle name */
+"CFBundleName" = "Extension AEM Sidekick";
```
Comment (kptdobe) [inline src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings:5]:
```suggestion
"CFBundleName" = "AEM Sidekick extension";
```

Same than above.

**Review by kptdobe — APPROVED** [review]

File: src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings (hunk 1 above)
Comment (rofe) [reply src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings:2]: Yes they are

**Pull request merged**